dds_cli/__main__.py

```diff
@@ -461,14 +461,14 @@ def rm(_, proj_arg, project, username, rm_all, file, folder):
 @click.option(
     "--username",
     "-u",
-    required=True,
+    required=False,
     type=str,
     help="Your Data Delivery System username.",
 )
 @click.option(
     "--project",
     "-p",
-    required=False,
+    required=True,
     type=str,
     help="Project ID to which you're uploading data.",
```

Comment:
Can you also change this: `to` --> `from` and `uploading` --> `downloading`

Reply:
Fixed!